src/overlays/actors/ovl_En_Guest/z_en_guest.c

```diff
@@ -20,23 +34,200 @@ const ActorInit En_Guest_InitVars = {
     (ActorFunc)EnGuest_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Guest/EnGuest_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Guest/EnGuest_Destroy.s")
+static ColliderCylinderInit_Set3 colliderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 10, 60, 0, { 0, 0, 0 } },
+};
+
+static InitChainEntry initChain[] = {
```

Comment:
we want to start using the same naming scheme as data from the code files, where we prefix with s for local/static variables and g for global
```suggestion
static ColliderCylinderInit_Set3 sColliderInit = {
    { COLTYPE_UNK10, 0x00, 0x00, 0x39, COLSHAPE_CYLINDER },
    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
    { 10, 60, 0, { 0, 0, 0 } },
};

static InitChainEntry sInitChain[] = {
```